<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;
</string>
   </property>
   <widget class="QWidget" name="plot_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>U (PWM,  â€°)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Y_ref (rpm)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>800</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="value" stdset="0">
     <double>123.123000000000005</double>
    </property>
    <property name="intValue" stdset="0">
     <number>123</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>650</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="value" stdset="0">
     <double>123.123000000000005</double>
    </property>
    <property name="intValue" stdset="0">
     <number>123</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>1271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>22</height>
     </size>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>620</y>
      <width>961</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>9999</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>9999</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>9999</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>500</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>725</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Y (rpm)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>724</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="value" stdset="0">
     <double>123.123000000000005</double>
    </property>
    <property name="intValue" stdset="0">
     <number>123</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
